src/components/Composer.vue

```diff
@@ -351,6 +351,7 @@ export default {
 				keysMissing: [],
 			},
 			editorMode: 'html',
+			addShareLink: t('mail', 'Add share link from {productName} Files', { productName: OC.theme.name ?? 'Nextcloud' }),
```

Comment:
```suggestion
			addShareLink: t('mail', 'Add share link from {productName} Files', { productName: OC?.theme?.name ?? 'Nextcloud' }),
```

because that will also work if the global variable OC or any of the sub properties don't exit